src/utilities/logging.ts

```diff
@@ -2,13 +2,12 @@ import { getPinoOptions, type LoggingTarget } from '@relaycorp/pino-cloud';
 import env from 'env-var';
 import pino, { type Level, type Logger } from 'pino';
 
-const DEFAULT_APP_NAME = 'veraid-authority';
+const DEFAULT_APP_NAME = 'awala-endpoint-internet';
 
 export function makeLogger(): Logger {
   const logTarget = env.get('LOG_TARGET').asString();
-  const gatewayVersion = env.get('AUTHORITY_VERSION').required().asString();
```

Comment:
This shouldn't have been deleted. Whilst optional, the version is very important when troubleshooting -- we use it in GCP Error Reporting.

The only problem was the "AUTHORITY" bit. It should be something like `ENDPOINT_VERSION` here. We could event shorten it to "VERSION".

Make sure to restore this env var in `src/testUtils/envVars.ts`.